.buildkite/release-pipelines/publish-release.yml

```diff
@@ -1,21 +1,26 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+agents:
+  queue: "mac"
+
+env:
+  IMAGE_ID: $IMAGE_ID
+
 steps:
   - label: "Publish Release"
-    plugins:
-      - $CI_TOOLKIT
+    plugins: [$CI_TOOLKIT]
     command: |
-      echo '--- :robot_face: Use bot for git operations'
-      source use-bot-for-git
```

Comment:
I think I missed where we started using `use-bot-for-git`. Do the commands in `configure-git-for-release-management.sh` replace the need for that?

Reply:
Yeah, I already mentioned in my previous comment, but for now `use-bot-for-git` only works on Android repos. Se paaHJt-6EP-p2.
The other commands just replicate whatever the other release pipelines are doing.